src/components/map/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { createRef, useEffect, useState, useCallback } from 'react';
+import { NEU_BOUNDS, LOADER } from './constants.js';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const MapDiv = styled.div`
+  width: 100%;
+  height: 90vh; // in theory should be in a container so i can set it to 100%
```

Comment:
❓ Should this be minHeight then? I’d also say it’s fine to set height to 100% and just leave a note on the component saying it’s required that it have a parent component with a set height. Another option would be to pass the height as a prop.